install.sh

```diff
@@ -59,9 +59,25 @@ case $PATH in
         ;;
 esac
 
-printf "Install location [default: %s]: " "$default_bin"
-read -r bindir < /dev/tty
-bindir=${bindir:-$default_bin}
+while [[ $# -gt 0 ]]
+do
+    key="$1"
+    case "$key" in
```

Comment:
This should have another case to handle an unknown argument, or it'll loop endlessly.